corelib-media-storage/src/main/java/eu/europeana/corelib/service/impl/swift/SwiftMediaStorageClientImpl.java

```diff
@@ -0,0 +1,150 @@
+package eu.europeana.corelib.service.impl.swift;
+
+import eu.europeana.corelib.domain.MediaFile;
+import eu.europeana.corelib.service.MediaStorageClient;
+import org.apache.commons.io.IOUtils;
+import org.jclouds.ContextBuilder;
+import org.jclouds.io.Payload;
+import org.jclouds.openstack.swift.v1.SwiftApi;
+import org.jclouds.openstack.swift.v1.domain.SwiftObject;
+import org.jclouds.openstack.swift.v1.features.ContainerApi;
+import org.jclouds.openstack.swift.v1.features.ObjectApi;
+import org.jclouds.openstack.swift.v1.options.PutOptions;
+
+import java.io.IOException;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.jclouds.io.Payloads.newByteArrayPayload;
+
+
+/**
+ * Created by salexandru on 03.06.2015.
+ */
+public class SwiftMediaStorageClientImpl implements MediaStorageClient {
+    private final ObjectApi objectApi;
+
+    public SwiftMediaStorageClientImpl(SwiftConfiguration config) {
+        if (null == config) {
+            throw new IllegalArgumentException("Config cannot be null");
+        }
+
+        System.out.println("============================================");
+        System.out.println("Configuration parameter: ");
+        System.out.println(config.getAuthUrl());
+        System.out.println(config.getContainerName());
+        System.out.println(config.getIdentity());
+        System.out.println(config.getPassword());
+        System.out.println(config.getRegionName());
+        System.out.println(config.getTenantName());
+        System.out.println(config.getUserName());
+        System.out.println("============================================");
+
+
+        final SwiftApi swiftApi = ContextBuilder.newBuilder("openstack-swift")
+                                 .credentials(config.getIdentity(), config.getPassword())
+                                 .endpoint(config.getAuthUrl())
+                                 .buildApi(SwiftApi.class);
+
+        final ContainerApi containerApi = swiftApi.getContainerApi(config.getRegionName());
+
+        if (null == containerApi.get(config.getContainerName())) {
```

Comment:
try to always write, if (value == null) instead, it's more in readable order with what you are asking here.